components/test_coprocessor_plugin/example_plugin/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "example_plugin"
+name = "test_coprocessor_plugin"
```

Comment:
I think it's meant to be an example instead of a test.

Reply:
Will rename to "example_coprocessor_plugin".